client/components/destination-info.jsx

```diff
@@ -207,9 +209,7 @@ export default class DestinationInfo extends React.Component {
                   </Link>
                 </div>
                 <div className="col-3">
-                  <div className="circle red m-auto d-flex justify-content-center align-items-center">
-                    <i onClick={() => this.handleClickDelete(destinationInfo.destinationId)} handler="delete" className="fas fa-trash-alt fa-lg"></i>
-                  </div>
+                  <DeleteModal destinationId={destinationId} handleDelete={this.handleClickDelete}/>
```

Comment:
I add the destinationId and the handDelete method to call in the child component DeleteModal here